plugins/formal-verification/src/org/jetbrains/kotlin/formver/conversion/ContractDescriptionConversionVisitor.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2010-2023 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.formver.conversion
+
+import org.jetbrains.kotlin.contracts.description.*
+import org.jetbrains.kotlin.fir.contracts.description.ConeContractConstantValues
+import org.jetbrains.kotlin.fir.diagnostics.ConeDiagnostic
+import org.jetbrains.kotlin.fir.types.ConeKotlinType
+import org.jetbrains.kotlin.formver.scala.silicon.ast.Exp
+import org.jetbrains.kotlin.formver.scala.silicon.ast.Type
+
+class ContractDescriptionConversionVisitor : KtContractDescriptionVisitor<Exp, StmtConversionContext, ConeKotlinType, ConeDiagnostic>() {
+    override fun visitConstantDescriptor(
+        constantReference: KtConstantReference<ConeKotlinType, ConeDiagnostic>,
+        data: StmtConversionContext
+    ): Exp {
+        val retType = data.methodCtx.returnVar.type.viperType
+        return when (constantReference) {
+            ConeContractConstantValues.WILDCARD -> Exp.BoolLit(true)
```

Comment:
We're leaning towards importing the `Exp.*` classes directly (via a wildcard, see the discussion [here](https://github.com/jesyspa/kotlin/commit/0db25bcfd765727f640b345cb4c1930237ba2ff0#r124163516)), so you can just do `BoolLit(true)`.